generated/p2719.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="utf-8" />
```

Comment:
Let's get rid of this file entirely, I guess we should only have html files for papers we're publishing.